### DISCUSSION TITLE
What are PC specs required for Blockchain development?

### DISCUSSION THREAD

**Discussion by MuhammadAmmar24** [nehal2210/Blockchain-Engineer-Course#11] [Q&A] [answered]
Assalamulalikum sir, 
What are PC specs required for Blockchain development?

**Comment by nehal2210** [comment, marked as answer]
WalikumAssalam,
For the development, you need a simple PC, core I5 or I3,  8 GB Ram, and 256 to  500 GB  HardDisk enough.
It would be great if you have SSD.
